src/test/java/com/team4/testingsystem/services/impl/TestsServiceImplTest.java

```diff
@@ -81,12 +81,26 @@ void getByIdFail() {
 
     }
 
+    @org.junit.jupiter.api.Test
+    void getByUserIdUserNotFound() {
```

Comment:
It could never happen. You are taking id from the JwtTokenUtil, which is getting user details from spring security, after it verified the correctness of the data